src/interfaces/Client/IClientLock.ts

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { TryLockResponse } from "../../types/lock/TryLockResponse";
+import { UnlockResponse } from "../../types/lock/UnlockResponse";
+
+/**
+ * Interface to acquire lock and releasing lock
+ */
+export default interface IClientLock {
+  /**
+   * 
```

Comment:
Can we add more context, similar to https://github.com/dapr/dapr/issues/3549#issue-970288624?
Example,
```
// Required. The lock store name,e.g. `redis`.
  string store_name = 1;

  // Required. resource_id is the lock key. e.g. `order_id_111`
  // It stands for "which resource I want to protect"
  string resource_id = 2;

  // Required. lock_owner indicate the identifier of lock owner.
  // You can generate a uuid as lock_owner.For example,in **JavaScript**:
  //
  // req.LockOwner = uuid.New().String() // js example instead
  //
  // This field is per request, not per process, so it is different for each request,
  // which aims to prevent multi-thread in the same process trying the same lock concurrently.
```